public/client.css

```diff
@@ -0,0 +1,84 @@
+body {
+  background-color: #FFF;
+  font-family: 'FiraSans', sans-serif;
+}
+
+[data-hide="false"] {
+  display: block;
+}
+
+[data-hide="true"] {
+  display: none;
+}
+
+.generic-loader-wrap {
+  left: calc(50% - 50px);
+  position: relative;
+  margin-top: 15%;
+  width:100px;
+  height: 100px;
+  border-radius: 50%;
+}
+.generic-loader-wrap:before {
+  content: url(http://hoodiehq.github.io/hoodie-css/src/content_img/animals/low-profile-dog-1.png);
+  position: absolute;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 1.3em;
+  margin-left: -30px;
+  margin-top: -40px;
+  color: #FFF;
+}
```

Comment:
please add the empty lines and see my other comment. Otherwise it’s looking good 👍